mmdet/datasets/pipelines/loading.py

```diff
@@ -231,7 +231,7 @@ def __init__(self,
         self.with_seg = with_seg
         self.poly2mask = poly2mask
         self.file_client_args = file_client_args.copy()
-        self.file_client = None
+        self.file_client = mmcv.FileClient(**self.file_client_args)
```

Comment:
Maybe there is no need to modify it. Because if you don't use the mask, it won't be initialized.

Reply:
ok